root/edit/OpenEdits.js

```diff
@@ -24,9 +24,9 @@ const OpenEdits = ({
   pager,
   refineUrlArgs,
 }: Props): React$Element<typeof Layout> => (
-  <Layout fullWidth title={l('Open Edits')}>
+  <Layout fullWidth title={l('Open edits')}>
```

Comment:
It could be mistaken with the verb form:

```suggestion
  <Layout fullWidth title={lp('Open edits', 'noun')}>
```

Reply:
Ok, adding context for this and Edit types in a separate commit at the end since there's several appearances in different commits.